cvat/settings/base.py

```diff
@@ -40,6 +41,10 @@
 ALLOWED_HOSTS = os.environ.get('ALLOWED_HOSTS', 'localhost,127.0.0.1').split(',')
 INTERNAL_IPS = ['127.0.0.1']
 
+REDIS_HOST = os.getenv('CVAT_REDIS_HOST', 'localhost')
+REDIS_PORT = os.getenv('CVAT_REDIS_PORT', 6379)
+REDIS_PASSWORD = urllib.parse.quote(os.getenv('CVAT_REDIS_PASSWORD', ''))
```

Comment:
The password itself shouldn't be quoted; the quoting is only needed for embedding it into a URL.